### DISCUSSION TITLE
use without registration

### DISCUSSION THREAD

**Discussion by alex-maxime** [dailydotdev/daily#888] [Feature Requests]
A few month a go, it was possible to use daily.dev without register with an email to have all the feature.
is it possible to be able to see all topics without create an account.
To simplify, back to old daily.dev

**Comment by idoshamun** [comment]
Hi @Alex237,

The new signup requirement helps us ensure the security and privacy of our users' information and improve the overall user experience. Supporting anonymous users yields technical and product debt that holds us back from providing even more value to our users.

We understand that change can be difficult, but we hope you will appreciate the benefits. 🙏
